xbmc/interfaces/builtins/SkinBuiltins.cpp

```diff
@@ -102,6 +102,50 @@ static int SetAddon(const std::vector<std::string>& params)
   return 0;
 }
 
+
+/*! \brief Select and choose a value for a skin setting.
+ *  \param params The parameters.
+ *  \details params[0] = Names of skin settings.
+ */
+static int SkinSelect(const std::vector<std::string>& params)
+{
+  std::vector<std::pair<std::string, std::string>> settings;
+  
+  auto pDlgSelect = static_cast<CGUIDialogSelect*>(g_windowManager.GetWindow(WINDOW_DIALOG_SELECT));
+  pDlgSelect->Reset();
+  pDlgSelect->SetHeading(CVariant{atoi(params[0].c_str())});
+  
+  for (auto &param : params)
+  {
+    if (param.find('|') != std::string::npos)
+    {
+      std::vector<std::string> values = StringUtils::Split(param, '|');
+      std::string label = g_localizeStrings.Get(atoi(values[0].c_str()));
```

Comment:
It would be better you make sure you got only 2 elements in the values array before you access the second index.